spec/features/find/search/viewing_a_course_spec.rb

```diff
@@ -69,13 +69,29 @@
     end
   end
 
-  scenario 'user views school placements' do
-    given_there_is_a_findable_course
-    when_i_visit_the_course_page
-    when_i_click('View list of school placements')
-    then_i_should_be_on_the_school_placements_page
-    when_i_click("Back to #{@course.name} (#{course.course_code})")
-    then_i_should_be_on_the_course_page
+  scenario 'user sees no school placements in 2025' do
+    create(:recruitment_cycle, :next)
+    Timecop.travel(1.month.since(Find::CycleTimetable.find_opens(2025))) do
+      allow(Settings).to receive(:current_recruitment_cycle_year).and_return(2025)
+      given_there_is_a_findable_course
+      @provider.update(selectable_school: false)
```

Comment:
Can we keep within the style that these tests were written in, Given, when, then etc

https://martinfowler.com/bliki/GivenWhenThen.html

Reply:
Sure